Makefile

```diff
@@ -7,6 +7,9 @@ all:
 	go build -o nspawn-runc ./cmd/nspawn-runc
 	go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
 
+cnibridge:
+	go get -u github.com/containernetworking/plugins/plugins/main/bridge
```

Comment:
why do we have the flag `--strip-vendor` in the first place?

Also, `glide.lock` does not contain "github.com/containernetworking/plugins". It seems the vendoring and `glide.lock` need to be fixed as part of this PR.